portalnet/src/overlay_service.rs

```diff
@@ -1258,13 +1305,14 @@ where
         } else {
             String::with_capacity(0)
         };
-        let cid = self.utp_socket.cid(enr, false);
+        let cid: utp_rs::cid::ConnectionId<crate::discovery::UtpEnr> =
```

Comment:
Imports should happen at the top of the file unless there's a good reason not to